test/avocado/seleniumlib.py

```diff
@@ -126,8 +134,14 @@ def send_keys(self, element, text, clear=True):
             self.driver.execute_script('var ev = document.createEvent("Event"); ev.initEvent("change", true, false); arguments[0].dispatchEvent(ev);', element)
             self.driver.execute_script('var ev = document.createEvent("Event"); ev.initEvent("keydown", true, false); arguments[0].dispatchEvent(ev);', element)
 
-    def wait(self, method, text, baseelement, overridetry, fatal, cond):
+    def wait(self, method, text, baseelement, overridetry, fatal, cond, jscheck):
         """
+This function is only for internal purposes:
+    General function for waiting on generic element
+    You can use it in case no other methods are implemented in this lib,
+    but ideally, write own function, instead of using this one directly.
+    This function stores caller function to dict, for purposes, that element is
+    lost and had to be renewed -> self.element_wait_functions
```

Comment:
How about this:
This function stores caller function for this element to an internal dictionary, in case that element is lost and has to be renewed (-> self.element_wait_functions)